main.py

```diff
@@ -1,72 +1,27 @@
+from user_welcome import welcome_member_to_the_guild
```

Comment:
as our functionality is swelling we should keep main to capture events only and delegate logic external files